modules/oc-mirror-about.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="installation-oc-mirror-about_{context}"]
+= About the oc-mirror plug-in
+
+The oc-mirror OpenShift CLI (`oc`) plug-in allows you to mirror all required {product-title} content and custom images to your mirror registry by using a single tool. It provides the following features:
+
+* Provides a centralized method to mirror {product-title} releases, Operators, helm charts, and custom images.
```

Comment:
the additionalImages field can pull official/vendor images along with custom images. Can this be changed to `other images` or `arbitrary images` or something else?